src/components/MessagesList/MessagesList.vue

```diff
@@ -113,11 +120,8 @@ export default {
 			}
 			return groups
 		},
-	},
-
-	watch: {
-		token: function() {
-			this.onTokenChange()
+		hasCancelRequest() {
```

Comment:
With the other comment, you don't need this anymore

Reply:
Makes sense now, I created the computed because I was using a similar condition in different places. So not to repeat it! But now I use it only in getNewMessages